.gitignore

```diff
@@ -40,7 +40,7 @@ develop-eggs/
 dist/
 downloads/
 eggs/
-lib/
+#lib/ # Producing an error: can't add files within Svelte's $lib/ directory
```

Comment:
Yeah we probably need to check that out in a different PR

Reply:
Yeah; I'll do it when I updating the common skeleton stuff in the docs right now and we can sync it when 

I also really should make our standard project skeleton a Cookiecutter template that I can sync with Cruft; I have some parts of it locally and some in the Governance and Guidelines repo but it would be nice to have it properly automated. However, while I did attempt to do so once, the amount of variance and the number of files involved was enough that I never got close to finishing before I had to do other things.